### ISSUE TITLE
undefined reference to llhttp_get_method

### ISSUE THREAD

**Issue by markand** [nodejs/llhttp#164]
Hi,

It looks like a function is missing in export because in the symbol table this function isn't present.

```
$ nm /usr/local/lib/libllhttp.so | grep llhttp_get
0000000000007002 T llhttp_get_errno
0000000000007010 T llhttp_get_error_pos
0000000000007006 T llhttp_get_error_reason
```

However it's properly declared in llhttp.h ending up in undefined reference.

**Comment by fduncanh** [comment]
This is probably fixed by the recent re-release of v6.0.7 (originally the main branch was accidentally used for the release instead of he release branch)

**Comment by ShogunPanda** [comment]
Fixed by #175.
